--- sources/TerrariaClientWindows/SteelSeriesEngineWrapper/FullSerializer/Internal/fsArrayConverter.cs
+++ sources/TerrariaBuildable/SteelSeriesEngineWrapper/FullSerializer/Internal/fsArrayConverter.cs
@@ -44,7 +_,7 @@
 
 	public override fsResult TryDeserialize(fsData data, ref object instance, Type storageType)
 	{
-		fsResult success = fsResult.Success;
+		fsResult success = FullSerializer.fsResult.Success;
 		fsResult fsResult = (success += CheckType(data, fsDataType.Array));
 		if (fsResult.Failed)
 		{
