--- sources/TerrariaClientWindows/SteelSeriesEngineWrapper/FullSerializer/Internal/fsIEnumerableConverter.cs
+++ sources/TerrariaBuildable/SteelSeriesEngineWrapper/FullSerializer/Internal/fsIEnumerableConverter.cs
@@ -57,7 +_,7 @@
 	public override fsResult TryDeserialize(fsData data, ref object instance_, Type storageType)
 	{
 		IEnumerable enumerable = (IEnumerable)instance_;
-		fsResult success = fsResult.Success;
+		fsResult success = FullSerializer.fsResult.Success;
 		fsResult fsResult = (success += CheckType(data, fsDataType.Array));
 		if (fsResult.Failed)
 		{
