--- sources/TerrariaServerWindows/Terraria/Map/WorldMap.cs
+++ sources/TerrariaServerMac/Terraria/Map/WorldMap.cs
@@ -234,10 +_,16 @@
 	public static bool TryGetMapPath(PlayerFileData playerFileData, WorldFileData worldFileData, out string mapPath)
 	{
 		string text = playerFileData.Path.Substring(0, playerFileData.Path.Length - 4);
+		char directorySeparatorChar = Path.DirectorySeparatorChar;
-		mapPath = text + Path.DirectorySeparatorChar + worldFileData.MapFileName + ".map";
+		mapPath = text + directorySeparatorChar + worldFileData.MapFileName + ".map";
 		if (worldFileData.UseGuidAsMapName && !FileUtilities.Exists(mapPath, playerFileData.IsCloudSave))
 		{
-			mapPath = text + Path.DirectorySeparatorChar.ToString() + worldFileData.WorldId + ".map";
+			object[] obj = new object[4] { text, null, null, null };
+			directorySeparatorChar = Path.DirectorySeparatorChar;
+			obj[1] = directorySeparatorChar.ToString();
+			obj[2] = worldFileData.WorldId;
+			obj[3] = ".map";
+			mapPath = string.Concat(obj);
 		}
 		return FileUtilities.Exists(mapPath, playerFileData.IsCloudSave);
 	}
