--- sources/TerrariaClientWindows/Terraria/Map/MapHelper.cs
+++ sources/TerrariaClientMac/Terraria/Map/MapHelper.cs
@@ -3036,7 +_,9 @@
 		{
 			Utils.TryCreatingDirectory(text);
 		}
+		string text2 = text;
+		char directorySeparatorChar = Path.DirectorySeparatorChar;
-		text = text + Path.DirectorySeparatorChar + Main.ActiveWorldFileData.MapFileName + ".map";
+		text = text2 + directorySeparatorChar + Main.ActiveWorldFileData.MapFileName + ".map";
 		new Stopwatch().Start();
 		if (!Main.gameMenu)
 		{
