--- sources/TerrariaClientWindows/ReLogic/ReLogic.csproj
+++ sources/TerrariaClientLinux/ReLogic/ReLogic.csproj
@@ -9,10 +_,8 @@
     <RootNamespace>ReLogic</RootNamespace>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="FNA" />
     <Reference Include="Ionic.Zip.CF" />
-    <Reference Include="Microsoft.Xna.Framework" />
-    <Reference Include="Microsoft.Xna.Framework.Game" />
-    <Reference Include="Microsoft.Xna.Framework.Graphics" />
     <Reference Include="Newtonsoft.Json" />
     <Reference Include="SteelSeriesEngineWrapper" />
     <Reference Include="System" />
