--- sources/TerrariaClientWindows/Terraria/Initializers/AssetInitializer.cs
+++ sources/TerrariaClientLinux/Terraria/Initializers/AssetInitializer.cs
@@ -11,7 +_,6 @@
 using Terraria.GameContent;
 using Terraria.ID;
 using Terraria.IO;
-using Terraria.Testing;
 using Terraria.Utilities;
 
 namespace Terraria.Initializers;
@@ -23,7 +_,6 @@
 		AssetReaderCollection assetReaderCollection = new AssetReaderCollection();
 		assetReaderCollection.RegisterReader(new PngReader(services.Get<IGraphicsDeviceService>().GraphicsDevice), ".png");
 		assetReaderCollection.RegisterReader(new XnbReader(services), ".xnb");
-		assetReaderCollection.RegisterReader(new FxReader(services.Get<IGraphicsDeviceService>().GraphicsDevice), ".fx");
 		AsyncAssetLoader asyncAssetLoader = new AsyncAssetLoader(assetReaderCollection, 20);
 		asyncAssetLoader.RequireTypeCreationOnTransfer(typeof(Texture2D));
 		asyncAssetLoader.RequireTypeCreationOnTransfer(typeof(DynamicSpriteFont));
