--- decompiled/TerrariaServerWindows/ReLogic/Content/Sources/FileSystemContentSource.cs
+++ decompiled/TerrariaServerLinux/ReLogic/Content/Sources/FileSystemContentSource.cs
@@ -20,7 +_,9 @@
 	{
 		_basePath = Path.GetFullPath(basePath);
 		if (!_basePath.EndsWith("/") && !_basePath.EndsWith("\\")) {
+			string basePath2 = _basePath;
-			_basePath += Path.DirectorySeparatorChar;
+			char directorySeparatorChar = Path.DirectorySeparatorChar;
+			_basePath = basePath2 + directorySeparatorChar;
 		}
 		BuildNameToAbsolutePathDictionary();
 	}
