--- sources/TerrariaServerWindows/Terraria/IO/WorldFile.cs
+++ sources/TerrariaServerLinux/Terraria/IO/WorldFile.cs
@@ -672,7 +_,9 @@
 			{
 				for (int num = Main.worldPathName.Length - 1; num >= 0; num--)
 				{
-					if (Main.worldPathName.Substring(num, 1) == (Path.DirectorySeparatorChar.ToString() ?? ""))
+					string text = Main.worldPathName.Substring(num, 1);
+					char directorySeparatorChar = Path.DirectorySeparatorChar;
+					if (text == (directorySeparatorChar.ToString() ?? ""))
 					{
 						Utils.TryCreatingDirectory(Main.worldPathName.Substring(0, num));
 						break;
@@ -680,15 +_,15 @@
 				}
 			}
 			Main.ActiveWorldFileData = CreateMetadata((Main.worldName == "") ? "World" : Main.worldName, _isWorldOnCloud, Main.GameMode);
-			string text = (Main.AutogenSeedName ?? "").Trim();
+			string text2 = (Main.AutogenSeedName ?? "").Trim();
-			if (WorldFileData.TryApplyingCopiedSeed(text, playSound: true, out var _, out var seedTextIncludingSecrets, out var _))
+			if (WorldFileData.TryApplyingCopiedSeed(text2, playSound: true, out var _, out var seedTextIncludingSecrets, out var _))
 			{
 				Main.ActiveWorldFileData.SetSeed(seedTextIncludingSecrets);
 			}
 			else
 			{
-				text = Utils.TrimUserString(text, WorldFileData.MAX_USER_SEED_TEXT_LENGTH);
+				text2 = Utils.TrimUserString(text2, WorldFileData.MAX_USER_SEED_TEXT_LENGTH);
-				AWorldGenerationOption optionFromSeedText = WorldGenerationOptions.GetOptionFromSeedText(text);
+				AWorldGenerationOption optionFromSeedText = WorldGenerationOptions.GetOptionFromSeedText(text2);
 				WorldGenerationOptions.SelectOption(optionFromSeedText);
 				foreach (AWorldGenerationOption option in WorldGenerationOptions.Options)
 				{
@@ -697,13 +_,13 @@
 						option.Enabled = true;
 					}
 				}
-				if (optionFromSeedText != null || string.IsNullOrWhiteSpace(text))
+				if (optionFromSeedText != null || string.IsNullOrWhiteSpace(text2))
 				{
 					Main.ActiveWorldFileData.SetSeedToRandomWithCurrentEvents();
 				}
 				else
 				{
-					Main.ActiveWorldFileData.SetSeed(text);
+					Main.ActiveWorldFileData.SetSeed(text2);
 				}
 			}
 			if (WorldGen.GenerateWorld(Main.AutogenProgress))
