--- sources/TerrariaClientWindows/Terraria/Main.cs
+++ sources/TerrariaServerWindows/Terraria/Main.cs
@@ -70,6 +_,7 @@
 using Terraria.Map;
 using Terraria.Net;
 using Terraria.ObjectData;
+using Terraria.Server;
 using Terraria.Social;
 using Terraria.Testing;
 using Terraria.UI;
@@ -80,7 +_,7 @@
 
 namespace Terraria;
 
-public class Main : Game
+public class Main : Terraria.Server.Game
 {
 	public static class CurrentFrameFlags
 	{
@@ -6342,7 +_,7 @@
 		{
 			Map = new WorldMap(maxTilesX, maxTilesY);
 			Configuration.Load();
-			graphics = new GraphicsDeviceManager(this as Game);
+			graphics = new GraphicsDeviceManager(this as Microsoft.Xna.Framework.Game);
 			base.Content.RootDirectory = "Content";
 		}
 	}
