--- sources/TerrariaClientWindows/Terraria/Main.cs
+++ sources/TerrariaServerWindows/Terraria/Main.cs
@@ -69,6 +_,7 @@
 using Terraria.Map;
 using Terraria.Net;
 using Terraria.ObjectData;
+using Terraria.Server;
 using Terraria.Social;
 using Terraria.Testing;
 using Terraria.UI;
@@ -79,7 +_,7 @@
 
 namespace Terraria;
 
-public class Main : Game
+public class Main : Terraria.Server.Game
 {
 	public static class CurrentFrameFlags
 	{
@@ -4579,10 +_,6 @@
 			_windowMover.TryMovingToScreen(screenDeviceName);
 			ThrottleWhenInactive = configuration.Get("ThrottleWhenInactive", defaultValue: true);
 			UnpinFromCore0 = configuration.Get("UnpinFromCore0", defaultValue: false);
-			if (UnpinFromCore0 && Platform.IsWindows)
-			{
-				WindowsPerformanceDiagnostics.UnpinFromCore0();
-			}
 		}
 		if (!flag)
 		{
@@ -6284,7 +_,7 @@
 		{
 			Map = new WorldMap(maxTilesX, maxTilesY);
 			Configuration.Load();
-			graphics = new GraphicsDeviceManager(this as Game);
+			graphics = new GraphicsDeviceManager(this as Microsoft.Xna.Framework.Game);
 			base.Content.RootDirectory = "Content";
 		}
 	}
@@ -62398,7 +_,7 @@
 
 	private void DrawRainInMenu()
 	{
-		bool flag = base.IsActive;
+		bool isActive = base.IsActive;
 		Microsoft.Xna.Framework.Rectangle[] array = new Microsoft.Xna.Framework.Rectangle[6];
 		for (int i = 0; i < array.Length; i++)
 		{
@@ -62411,7 +_,7 @@
 			{
 				Rain rain = Main.rain[j];
 				spriteBatch.Draw(TextureAssets.Rain.Value, rain.position - screenPosition, array[rain.type], color, rain.rotation, Vector2.Zero, rain.scale, SpriteEffects.None, 0f);
-				if (flag)
+				if (isActive)
 				{
 					rain.Update();
 				}
