--- sources/TerrariaClientWindows/Terraria/Terraria.csproj
+++ sources/TerrariaServerWindows/Terraria/Terraria.csproj
@@ -6,26 +_,21 @@
     <Version>1.4.5.4</Version>
     <Company>Re-Logic</Company>
     <Copyright>Copyright © 2026 Re-Logic</Copyright>
-    <RootNamespace>Terraria</RootNamespace>
+    <RootNamespace>TerrariaServer</RootNamespace>
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="CsvHelper" />
     <Reference Include="Ionic.Zip.CF" />
     <Reference Include="Microsoft.Xna.Framework" />
-    <Reference Include="Microsoft.Xna.Framework.Content.Pipeline" />
     <Reference Include="Microsoft.Xna.Framework.Game" />
     <Reference Include="Microsoft.Xna.Framework.Graphics" />
     <Reference Include="Microsoft.Xna.Framework.Xact" />
-    <Reference Include="MP3Sharp" />
     <Reference Include="Newtonsoft.Json" />
-    <Reference Include="NVorbis" />
     <ProjectReference Include="../RailSDK.Net/RailSDK.Net.csproj" />
     <EmbeddedResource Include="../RailSDK.Net/bin/$(Configuration)/$(TargetFramework)/RailSDK.Net.dll" />
     <ProjectReference Include="../ReLogic/ReLogic.csproj" />
     <EmbeddedResource Include="../ReLogic/bin/$(Configuration)/$(TargetFramework)/ReLogic.dll" />
     <Reference Include="Steamworks.NET" />
-    <ProjectReference Include="../SteelSeriesEngineWrapper/SteelSeriesEngineWrapper.csproj" />
-    <EmbeddedResource Include="../SteelSeriesEngineWrapper/bin/$(Configuration)/$(TargetFramework)/SteelSeriesEngineWrapper.dll" />
     <Reference Include="System" />
     <Reference Include="System.Core" />
     <Reference Include="System.Drawing" />
@@ -40,6 +_,5 @@
     <EmbeddedResource Include="IO/Data/*" />
     <EmbeddedResource Include="Libraries/**" />
     <EmbeddedResource Include="Localization/Content/**" />
-    <EmbeddedResource Include="Microsoft.Xna.Framework.RuntimeProfile" />
   </ItemGroup>
 </Project>
