--- sources/TerrariaServerWindows/Terraria/Localization/LanguageManager.cs
+++ sources/TerrariaServerMac/Terraria/Localization/LanguageManager.cs
@@ -180,7 +_,8 @@
 	public void UseSources(List<IContentSource> sourcesFromLowestToHighest)
 	{
 		string name = ActiveCulture.Name;
+		char directorySeparatorChar = Path.DirectorySeparatorChar;
-		string assetNameStart = ("Localization" + Path.DirectorySeparatorChar + name).ToLower();
+		string assetNameStart = ("Localization" + directorySeparatorChar + name).ToLower();
 		LoadLanguage(ActiveCulture, processCopyCommands: false);
 		foreach (IContentSource item in sourcesFromLowestToHighest)
 		{
